L104-core-ban-recv-with-send-status.md

```diff
@@ -0,0 +1,36 @@
+Title
+----
+* Author(s): ctiller
+* Approver: markdroth
+* Status: In Review
+* Implemented in: C Core
+* Last updated: 2022/11/03
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Please create the thread and add a link here.

Reply:
done